dashboard/test/ui/features/curriculum_platform/levelbuilder/lesson_edit_page.feature

```diff
@@ -0,0 +1,40 @@
+# We need "press keys" to type into the React form's fields, but that doesn't work on IE.
+@no_ie
+@no_mobile
+Feature: Using the Lesson Edit Page
+  Scenario: Navigate from script edit page to lesson edit page
+    Given I create a levelbuilder named "Levi"
+    And I create a temp script and lesson
+    And I view the temp script overview page
+    And I view the temp script gui edit page
```

Comment:
Should the script_edit_page.feature test the script gui page too?

Reply:
That seems sensible to me. do you think this one should live there, then?

Reply:
Ya probably